Chore/backport fixes

khawkins98 opened the pull request:
This picks up a bunch of the low hanging fruit fixes for vf-core components that were in the local static-html-pages repo; see https://gitlab.ebi.ac.uk/emblorg/static-html-pages/merge_requests/156

It also would synchronise our little production environment with vf-core, which would be nice.

khawkins98 committed:
Chore: backport fixes from static-html-pages

This picks up a bunch of the low hanging fruit fixes for vf-core components that were in the local static-html-pages repo; see https://gitlab.ebi.ac.uk/emblorg/static-html-pages/merge_requests/156

khawkins98 committed:
Linter

khawkins98 committed:
Add changelog

sturobson commented:
There's also this - https://gitlab.ebi.ac.uk/emblorg/static-html-pages/blob/sr/overhaul-homepage/src/components/emblstatic-homepage/emblstatic-homepage.scss#L182 - from a branch in `static-html-pages` that I need to work on still

khawkins98 committed:
Update vf-card-container.scss

Backports https://gitlab.ebi.ac.uk/emblorg/static-html-pages/blob/sr/overhaul-homepage/src/components/emblstatic-homepage/emblstatic-homepage.scss#L182

khawkins98 commented:
Makes sense to go ahead and bake that in; seem right? https://github.com/visual-framework/vf-core/pull/760/commits/a06fcb09d237f572e66af1f173412318fa6aaec9

khawkins98 committed:
Fix width

khawkins98 commented:
Missed a line: https://github.com/visual-framework/vf-core/pull/760/commits/734fb95af6cd25b47bef989629385b20b5ae9281
